src/core/reactor.cc

```diff
@@ -891,6 +892,13 @@ static decltype(auto) install_signal_handler_stack() {
         }
     });
 }
+#else
+// SIGSTKSZ is too small when using asan. We also don't need to
+// handle SIGSEGV ourselves when using asan, so just don't install
+// a signal handler stack.
+struct nothing { ~nothing() {} };
+nothing install_signal_handler_stack() { return nothing{}; }
```

Comment:
nit, could move `nothing` into the body of `install_signal_handler_stack()`, and make the function to return `auto`, so the symbol of `nothing` is not exposed, and we don't need to move it into an anonymous namespace.

Reply:
Hm, good point, will try

Reply:
Yup, it did the trick, thanks!